jaseci_ai_kit/jaseci_ai_kit/modules/encoders/tests/train_data.json

```diff
@@ -0,0 +1,194 @@
+{
```

Comment:
We can just do 2 intents, each with 3 utterances. and train for 2 epochs.

Reply:
done